docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,96 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory service used for prototyping and development purpose. You can find the code for the [Tinylicious service](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious) and [Tinylicious client](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client). It can be instantiated against both `AzureClient` and `TinyliciousClient` for testing purpose.
```

Comment:
```suggestion
Tinylicious is a local, in-memory Fluid service intended for prototyping and development purpose. You can use both [AzureClient]]({{< relref "azureclient.md" >}}) and [TinyliciousClient]({{< relref "tinyliciousclient.md" >}}) with Tinylicious for testing purposes.
```
I'd omit the links to the source; if they're important add them to a "more information" section at the bottom of the doc.